HMCL/src/main/java/org/jackhuang/hmcl/ui/versions/VersionPage.java

```diff
@@ -322,10 +335,24 @@ protected Skin(VersionPage control) {
                         new IconedMenuItem(null, i18n("version.manage.clean"), control::clearJunkFiles, managementPopup).addTooltip(i18n("version.manage.clean.tooltip"))
                 );
 
+                AdvancedListItem pinVersionItem = new AdvancedListItem();
+                pinVersionItem.getStyleClass().add("navigation-drawer-item");
+                pinVersionItem.setLeftGraphic(wrap(SVG.THUMBTACK));
+
+                pinVersionItem.titleProperty().bind(
+                        Bindings.createStringBinding(
+                                () -> control.currentVersionPinned.get() ? i18n("version.unpin") : i18n("version.pin"),
+                                control.currentVersionPinned
+                        )
+                );
+                pinVersionItem.activeProperty().bind(control.currentVersionPinned);
+                pinVersionItem.setOnAction(e -> control.pinVersion(pinVersionItem));
+
                 AdvancedListBox toolbar = new AdvancedListBox()
                         .addNavigationDrawerItem(i18n("version.update"), SVG.UPDATE, control::updateGame, upgradeItem -> {
                             upgradeItem.visibleProperty().bind(control.currentVersionUpgradable);
                         })
+                        .add(pinVersionItem)
```

Comment:
```suggestion
                AdvancedListBox toolbar = new AdvancedListBox()
                        .addNavigationDrawerItem(i18n("version.update"), SVG.UPDATE, control::updateGame, upgradeItem -> {
                            upgradeItem.visibleProperty().bind(control.currentVersionUpgradable);
                        })
                        .addNavigationDrawerItem("",SVG.THUMBTACK,control::pinVersion,pinVersionItem->{
                            pinVersionItem.titleProperty().bind(
                                    Bindings.createStringBinding(
                                            () -> control.currentVersionPinned.get() ? i18n("version.unpin") : i18n("version.pin"),
                                            control.currentVersionPinned
                                    )
                            );
                            pinVersionItem.activeProperty().bindBidirectional(control.currentVersionPinned);
                        })
```
感觉可以改成这样写，写法更统一并且可以解决高度问题